### PR TITLE
Update SQL Transport

### PR DESCRIPTION
**Pull request by mikeminutillo**
Waiting on release of 2.2.5 to fix SQL version of #911

### CODE REVIEW

**Comment by SzymonPobiega** [comment]
@mikeminutillo 2.2.5 is out on NuGet

File: src/ServiceControl.AcceptanceTests/AcceptanceTest.cs [hunk 1]
```diff
@@ -103,9 +106,26 @@ public void Setup()
             ravenPath = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName());
         }
 
+        private void AssertTransportNotExplicitlyIgnored()
+        {
+            var key = (object) "IgnoreTransports";
```
Comment (SzymonPobiega) [inline src/ServiceControl.AcceptanceTests/AcceptanceTest.cs]: why do we need a cast?

**Comment by SzymonPobiega** [comment]
@mikeminutillo is it still WIP?

**Comment by mikeminutillo** [comment]
Not still WIP. Happy to remove the cast before merging. It used to be a different key and for some reason the dictionary is (object) -> (object)

**Pull request merged**